### DISCUSSION TITLE
Unknown issue Exit status 65

### DISCUSSION THREAD

**Discussion by nhatduy129** [fastlane/fastlane#18440] [Q&A]
I am trying to build my project by fastlane, but it does not work.
It returns to me error below. Is there any one have ideas? Many thanks.
<img width="1788" alt="Screenshot 2021-03-23 at 4 41 17 PM" src="https://user-images.githubusercontent.com/11719495/112126002-98667800-8bf6-11eb-8761-984961db0a4c.png">

**Comment by vandat0599** [comment]
I'm having the same issue. Thanks, @nhatduy129.

**Reply by bestmmmz** [reply]
same issue
help

**Comment by ptmkenny** [comment]
I'm having the same issue.  Here's what I know so far.

First, the problem is caused by the archive step failing.  You should be able to reproduce the problem in XCode by building the project and then selecting `Product` -> `Archive` from the menu.  This will fail with the same error.

So the problem is that Fastlane can't archive because it is using XCode to do the archiving, and XCode is failing.

The question is: how to fix this?

There are three issues in the issue queue:
* https://github.com/fastlane/fastlane/issues/8638 suggests to check the signing config.  In my case, I have signing set up as instructed.
* https://github.com/fastlane/fastlane/issues/1005 says to use `use_legacy_build_api`, but this seems to be no longer available.
* https://github.com/fastlane/fastlane/issues/6456 notes to check the XCode errors carefully and fix them.

I'm still debugging but hopefully this helps to find a solution.

**Reply by ptmkenny** [reply]
In my case, the cause was Firebase Crashlytics being incorrectly configured in a build script.  The XCode GUI did not report an error, but it was failing when built via Fastlane, which uses a command-line tool slightly different than the XCode GUI.

**Comment by Carissa34** [comment]
We ran "xcodebuild" command but it exited with error code 65.” docker.service: Failed with result 'exit-code'

https://www.myindigocard.bid/

**Comment by Keon241** [comment]
I found the cause of the problem I was having: the add-on product's testing code was foolishly calling C's exit(0) when it though it had finished. However, this is a very slow way to build a list of exit values: a published list would be very useful.

[MyHerbalife Login](https://www.myherbalife.life/)
